src/GovUk.Education.ExploreEducationStatistics.Admin/Services/DataBlockService.cs

```diff
@@ -301,17 +306,19 @@ private Release GetReleaseForDataBlock(Guid dataBlockId)
                 .OnSuccess(_ => dataBlock);
         }
     }
-    
+
     public class DependentDataBlock
     {
         [JsonIgnore]
         public Guid Id { get; set; }
-        
+
         public string Name { get; set; }
-        
+
         public string? ContentSectionHeading { get; set; }
-        
+
         public List<string> InfographicFilenames { get; set; }
+
+        public List<Guid> InfographicFileIds { get; set; }
```

Comment:
Having two separate lists is a bit messy. Could group these filename and id together as a nested `InfographicFileInfo` or something, so the end type would be `List<InfographicFileInfo>`.

Reply:
done